pkg/providers/nutanix/config/cp-template.yaml

```diff
@@ -0,0 +1,196 @@
+  controlPlaneRef:
+    apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+    kind: KubeadmControlPlane
+    name: "{{.clusterName}}"
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+    kind: NutanixCluster
+    name: "{{.clusterName}}"
+---
+apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+kind: KubeadmControlPlane
+metadata:
+  name: "{{.clusterName}}"
+  namespace: "{{.eksaSystemNamespace}}"
+spec:
+  replicas: {{.controlPlaneReplicas}}
+  version: "{{.kubernetesVersion }}"
+  machineTemplate:
+    infrastructureRef:
+      apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+      kind: NutanixMachineTemplate
+      name: "{{.clusterName}}-mt-0"
+  kubeadmConfigSpec:
+    clusterConfiguration:
+      imageRepository: "{{.kubernetesRepository}}"
+      apiServer:
+        certSANs:
+          - localhost
+          - 127.0.0.1
+          - 0.0.0.0
+      controllerManager:
+        extraArgs:
+          enable-hostpath-provisioner: "true"
+      dns:
+        imageRepository: {{.corednsRepository}}
+        imageTag: {{.corednsVersion}}
+      etcd:
+  {{- if .externalEtcd }}
+external:
+  endpoints: []
+  caFile: "/etc/kubernetes/pki/etcd/ca.crt"
+  certFile: "/etc/kubernetes/pki/apiserver-etcd-client.crt"
+  keyFile: "/etc/kubernetes/pki/apiserver-etcd-client.key"
+  {{- else }}
+local:
+  imageRepository: {{.etcdRepository}}
+  imageTag: {{.etcdImageTag}}
+  {{- end }}
+files:
+  - content: |
+      apiVersion: v1
+      kind: Pod
+      metadata:
+        creationTimestamp: null
+        name: kube-vip
+        namespace: kube-system
+      spec:
+        containers:
+          - name: kube-vip
+            image: {{.kubeVipImage}}
+            imagePullPolicy: IfNotPresent
+            args:
+              - manager
+            env:
+              - name: vip_arp
+                value: "true"
+              - name: address
+                value: "{{.controlPlaneEndpointIp}}"
+              - name: port
+                value: "6443"
+              - name: vip_cidr
+                value: "32"
+              - name: cp_enable
+                value: "true"
+              - name: cp_namespace
+                value: kube-system
+              - name: vip_ddns
+                value: "false"
+              - name: vip_leaderelection
+                value: "true"
+              - name: vip_leaseduration
+                value: "15"
+              - name: vip_renewdeadline
+                value: "10"
+              - name: vip_retryperiod
+                value: "2"
+            securityContext:
+              capabilities:
+                add:
+                  - NET_ADMIN
+                  - SYS_TIME
+                  - NET_RAW
+            volumeMounts:
+              - mountPath: /etc/kubernetes/admin.conf
+                name: kubeconfig
+            resources: {}
+        hostNetwork: true
+        hostAliases:
+          - hostnames:
+              - kubernetes
+            ip: 127.0.0.1
```

Comment:
We patch the `kube-vip` we use [here](https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/kube-vip/kube-vip/patches/0001-use-hostname-instead-of-kubernetes-to-contact-apiser.patch) so this isn't needed